src/game.js

```diff
@@ -73,4 +78,19 @@ export class Game
 
         this.app.state.scene.value.update(deltaTime);
     }
+
+    resize(event)
+    {
+        const game = this.game;
+
+        var w = event.width;
```

Comment:
Is there a reason this is written like this?
Could we not write it like this?
```js
const { width, height } = event;
const scale = Math.min(width / GAMEPLAY.WIDTH, height / GAMEPLAY.HEIGHT);
this.pixi.renderer.view.setAttribute('style',  `-ms-transform: scale(${scale}); -webkit-transform: scale3d(${scale}, 1); -moz-transform: scale(${scale}); -o-transform: scale(${scale}); transform: scale(${scale});
 transform-origin: top left;`);
```

Reply:
That was an artifact from some experimenting. I cleaned it up a bit.